binnacle/lib/main.dart

```diff
@@ -1,137 +1,46 @@
 import 'package:flutter/material.dart';
-import 'providers/app_provider.dart';
-import 'screens/test_screen.dart';
-// import 'package:flutter/services.dart';
-// import 'package:intl/intl.dart';
-
-// import './CoordInputRoute.dart'; //ignore: unused_import
-// import './model/DataModel.dart';
-// import './BinnacleWidget.dart';
-
-// import './ui/deck/DeckWidget.dart'; //ignore: unused_import
+import 'package:flutter/services.dart';
 
-// import './model/bluetooth/BluetoothManager.dart'; //ignore: unused_import
-
-void main() => runApp(MyApp());
+import 'package:sos/ui/list_angle_widget.dart';
+import 'providers/app_provider.dart';
+import 'ui/info_panel.dart';
+import 'ui/compass/BinnacleBase.dart';
+import './ui/global_theme.dart';
+import './ui/app_drawer.dart';
+
+void main() {
+  SystemChrome.setPreferredOrientations([DeviceOrientation.portraitUp]);
+  runApp(MyApp());
+}
 
 class MyApp extends StatelessWidget {
   // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
     return Provider(
-      child: MaterialApp(
-        title: 'Binnacle Demo',
-        theme: ThemeData(
-        // This is the theme of your application.
-        //
-        // Try running your application with "flutter run". You'll see the
-        // application has a blue toolbar. Then, without quitting the app, try
-        // changing the primarySwatch below to Colors.green and then invoke
-        // "hot reload" (press "r" in the console where you ran "flutter run",
-        // or simply save your changes to "hot reload" in a Flutter IDE).
-        // Notice that the counter didn't reset back to zero; the application
-        // is not restarted.
-          primarySwatch: Colors.blueGrey,
-        ),
-        home: TestScreen()
-      )
-      
-      // home: MyHomePage(title: 'Binnacle Demo (Main)'),
-    );
+        child: MaterialApp(
+            title: 'Binnacle Demo',
+            theme: new GlobalTheme().get(),
```

Comment:
This was one of my main reasons for doing the theme in its own file so that any UI component can operate independent of their parents. Helps with testing